go/vt/vttablet/tabletserver/connpool/pool.go

```diff
@@ -109,7 +113,15 @@ func (cp *Pool) Open(appParams, dbaParams, appDebugParams dbconfigs.Connector) {
 	f := func(ctx context.Context) (pools.Resource, error) {
 		return NewDBConn(ctx, cp, appParams)
 	}
-	cp.connections = pools.NewResourcePool(f, cp.capacity, cp.capacity, cp.idleTimeout, cp.prefillParallelism, cp.getLogWaitCallback())
+
+	var refreshCheck pools.RefreshCheck
+	if net.ParseIP(appParams.Host()) == nil {
+		refreshCheck = netutil.DNSTracker(appParams.Host())
+	} else {
+		refreshCheck = nil
+	}
```

Comment:
nit: we can remove this `else` statement

Reply:
yup.  For some reason my IDE insisted on this check;  I must have had some setting enabled.  I will yank it.